core/unit_test/CMakeLists.txt

```diff
@@ -64,7 +62,8 @@ macro(Cabana_add_tests)
       )
       set(_target ${_test}_test_${_device})
       add_executable(${_target} ${_file} ${CABANA_UNIT_TEST_MAIN})
-      target_include_directories(${_target} PUBLIC ${_dir})
+      target_include_directories(${_target} PUBLIC ${_dir}
```

Comment:
It seems `PUBLIC` doesn't make sense for executables (https://reviews.llvm.org/D40823).